dropwizard-cassandra/src/main/java/smartthings/dw/cassandra/CassandraConfiguration.java

```diff
@@ -309,4 +323,106 @@ private static SSLContext getSSLContext(String truststorePath, String truststore
 		ctx.init(kmf.getKeyManagers(), tmf.getTrustManagers(), new SecureRandom());
 		return ctx;
 	}
+
+    public static class PoolConfiguration {
+        private Integer idleTimeoutSeconds;
+        private Integer heartbeatIntervalSeconds;
+        private Integer maxQueueSize;
+        private Integer poolTimeoutMillis;
+        private Integer localMaxConnPerHost;
+        private Integer remoteMaxConnPerHost;
```

Comment:
Used `Integer` to allow for null and preserve default driver behavior where needed.